paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -168,17 +181,56 @@ def http_metrics_provider(marathon_service_config, marathon_tasks, mesos_tasks,
     utilization = []
     for task in marathon_tasks:
         try:
-            endpoint_utilization = requests.get(
-                'http://%s:%s/%s' % (task.host, task.ports[0], endpoint),
-                headers={'User-Agent': get_user_agent()}
-            ).json()['utilization']
-            utilization.append(float(endpoint_utilization))
+            utilization.append(json_mapper(get_json_body_from_service(task.host, task.ports[0], endpoint)))
         except Exception:
             pass
```

Comment:
Right, but that is the problem:
* Expected: That when all workers are full the autoscaler scales things up
* Actual: When all workers are full the autoscaler raises and doesn't scale anything